worlds/witness/__init__.py

```diff
@@ -178,14 +178,14 @@ def create_items(self):
                     item_pool[random_early_item] -= 1
 
         # Generate the actual items.
+
+        item_pool = {k: v for k, v in sorted(item_pool.items(), key=lambda item: item[0])}
```

Comment:
this sorts by key, right?

are we sure this comprehension is better than simply doing `dict(sorted(item_pool.items())`

Reply:
That gave me a lint error for some reason

Reply:
```Unexpected type(s): (Generator[list[str], Any, None]) Possible type(s): (SupportsKeysAndGetItem[list[str], _VT]) (Iterable[tuple[Any, Any]])```

Reply:
is it possible that you tried `sorted(item_pool)` instead of `sorted(item_pool.items())` ? sorted(item_pool) would *just* be the keys.